packages/hash/api/src/graphql/resolvers/org/createOrg.ts

```diff
@@ -9,11 +9,13 @@ export const createOrg: Resolver<
   MutationCreateOrgArgs
 > = async (_, { org: orgInput, responsibility }, { dataSources, user }) =>
   dataSources.db.transaction(async (client) => {
+    const { graphApi } = dataSources;
     const { shortname, name, orgSize } = orgInput;
 
-    await user
+    /** @todo: potentially deprecate these method calls depending on Graph API transaction implementation */
+    await (user as any)
       .acquireLock(client)
-      .then(() => user.refetchLatestVersion(client));
+      .then(() => (user as any).refetchLatestVersion(client));
```

Comment:
Why does the users here all need to be cast as any? Is it because of the way `LoggedInGraphQLContext` extends `GraphQLContext`? I have never seen that `default` type it is set as in the editor when I mouseover the non-casted `user`

Reply:
I've changed the type of `user` in `LoggedInGraphQLContext` to have the updated `UserModel` as its value. `UserModel` doesn't implement `refetchLatestVersion`, which is why I'm casting here to temporarily suppress this linting error.

You are seeing `default` instead of `UserModel` when you hover over the variable in your code editor because we are default exporting un-named classes in the model classes. I think the code editor only tried to look up the name of the class where it's defined, and therefore can't get any better name than `default`.